code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -358,6 +358,10 @@
 	C.slowdown = 0
 	qdel(src)
 
+/obj/item/slimepotion/speed/MouseDrop(obj/over_object)
+	if(loc == usr && loc.Adjacent(over_object))
```

Comment:
you could do this while incapped.

Reply:
wait what do you mean im confused

Reply:
there's no checks for if you're dead, stunned, or what have you.

Reply:
K I fixed it (commenting for the noti)